src/hydra/hydraClient.js

```diff
@@ -168,8 +168,9 @@ export default ({entrypoint, resources = []}, httpClient = fetchHydra) => {
    * @returns {Object}
    */
   const convertReactAdminRequestToHydraRequest = (type, resource, params) => {
-    const collectionUrl = new URL(`${entrypoint}/${resource}`);
-    const itemUrl = new URL(params.id, entrypoint);
+    const entrypointUrl = new URL(entrypoint, window.location.origin);
```

Comment:
We could even use `window.location.href` to support relative paths (as in your test)

Reply:
I don't see any difference between window.location.origin and window.location.href, except that window.location.href would include potential anchors (https://domain#foo), but I think they are treated the same way by the URL class.

Reply:
I don't see any difference between window.location.origin and window.location.href, except that window.location.href would include potential anchors (https://domain#foo), but I think they are treated the same way by the URL class.

Reply:
```
> window.location.href
< "https://github.com/api-platform/admin/pull/126"
> window.location.origin
< "https://github.com"
```

With `window.location.href`, relative path will work.

Reply:
```
> new URL('foo/bar', window.location.origin)
< URL {href: "https://github.com/foo/bar", origin: "https://github.com", protocol: "https:", username: "", password: "", …}
> new URL('foo/bar', window.location.href)
< URL {href: "https://github.com/api-platform/admin/pull/foo/bar", origin: "https://github.com", protocol: "https:", username: "", password: "", …}
```

Reply:
Right, sorry, my bad. I was only testing in the case where the base URL is the origin...

```
> new URL('foo/bar', (new URL('https://github.com/api-platform')).origin)
< URL { href: "https://github.com/foo/bar" }
> new URL('foo/bar', (new URL('https://github.com/api-platform')).href)
< URL { href: "https://github.com/foo/bar" }
```

Reply:
Wow, it looks like a browser bug to me. The origin should never include the path.

Reply:
It doesn't, my example is just the special case where the path length is 1. In that case the base URL is the origin.